package.json

```diff
@@ -107,8 +108,6 @@
   },
   "devDependencies": {
     "@babel/core": "^7.20.12",
-    "@netlify/esbuild": "0.14.39",
-    "@netlify/plugin-nextjs": "^4.38.1",
```

Comment:
The `@netlify/plugin-nextjs` is not required, as mentioned in #1482. I'm curious why `@netlify/esbuild` was being imported though. 🤔